cssom/Overview.bs

```diff
@@ -256,6 +256,41 @@ the last item. Unless otherwise specified, an empty list is serialized as the
 empty string.
 
 
+CSSOMString {#cssomstring-type}
+===============================
+
+Most strings in CSSOM interfaces use the <dfn>CSSOMString</dfn> type.
+Each implementation chooses to define it as either {{USVString}} or {{DOMString}}:
+
+<pre class=idl>
+typedef USVString CSSOMString;
+</pre>
+
+Or, alternatively:
+
+<pre class="def lang-webidl">
```

Comment:
I haven't added the "extract" support yet.  (The bug is still open!)

(And in any case, I think it's probably better to do exactly this - instead of claiming something is WebIDL and then opting out, just ask it to be highlighted as WebIDL but otherwise be meaningless text.)